lib/Service/LocalProjectService.php

```diff
@@ -325,8 +326,10 @@ public function deleteProject(string $projectId): void {
 	 * @throws DoesNotExistException
 	 * @throws MultipleObjectsReturnedException
 	 * @throws \OCP\DB\Exception
+	 * @throws AppConfigTypeConflictException
 	 */
 	public function getProjectInfo(string $projectId): array {
+		$this->memberMapper->cleanupUserMembers($projectId);
```

Comment:
If you still want to do this, then it would be better to move it after you have queried the project, as the next lines will handle the error if it doesn't exist so the cleanup won't have to run for projects that don't exist.

Reply:
Nope, I just forgot to remove it.